dynawo/sources/Models/Modelica/Dynawo/Examples/Photovoltaics/WECC/PVCurrentSource.mo

```diff
@@ -0,0 +1,57 @@
+within Dynawo.Examples.Photovoltaics.WECC;
```

Comment:
this example makes V vary from 1 to 0.5 Pu, i remember you said you wanted to try a threephase fault with no resistance

Reply:
I have some data missing on the reference side in the nrt results. Did you add the reference files? 
otherwise, only two displayed curves Pinj and Qinj, don't we want i and u at terminal too in the example?

Reply:
Yes you are right, but the frank fault doesn't work on dynawo so I keep this also in OM.
Yes, the missing data are the logs, which are not necessary. We could add more curves yes.